### ISSUE TITLE
gtmOnSuccess and gtmOnFailure must be called once and only once

### ISSUE THREAD

**Issue by semisight** [adless-tech/google-tag-manager-no-ads#1]
https://github.com/adless-tech/google-tag-manager-no-ads/blob/6dcea8d2065a0ac112713256d44b231f8e9ad505/template.tpl#L134

This method calls gtmOnFailure, but never calls gtmOnSuccess at the end. This is likely to be a bug.

**Comment by MathiasGrund** [comment]
The call is actually superfluous; removed it entirely as gtmOnXXX should only be handled in onRejectServices, which is always called.

**Comment by MathiasGrund** [comment]
@semisight This issue was resolved 3 weeks ago. What is the procedure to get a fresh review for admission to the community gallery?

**Comment by semisight** [comment]
Apologies for leaving this waiting so long. Your template has been merged into gallery and should be available soon.
